src/libraries/System.Net.Security/src/System/Net/NegotiateAuthenticationPal.ManagedNtlm.cs

```diff
@@ -424,10 +424,23 @@ private unsafe void WriteChannelBindingHash(Span<byte> hashBuffer)
             {
                 if (_channelBinding != null)
                 {
-                    IntPtr cbtData = _channelBinding.DangerousGetHandle();
-                    int cbtDataSize = _channelBinding.Size;
-                    int written = MD5.HashData(new Span<byte>((void*)cbtData, cbtDataSize), hashBuffer);
-                    Debug.Assert(written == MD5.HashSizeInBytes);
+                    int secChannelBindingsSize = Marshal.SizeOf<SecChannelBindings>();
```

Comment:
```suggestion
                    int secChannelBindingsSize = sizeof(SecChannelBindings)
```

Reply:
I'll change it both here and in SafeChannelBindingHandle then.